src/filter/defish0r/defish0r.c

```diff
@@ -394,6 +477,21 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
 		info->type = F0R_PARAM_DOUBLE;
 		info->explanation = "Manual Pixel Aspect ratio";
 		break;
+	case 8:
+		info->name = "Crop";
+		info->type = F0R_PARAM_BOOL;
+		info->explanation = "Straighten all edges of video frame";
+		break;
+	case 9:
+		info->name = "Dynamic Stretch";
```

Comment:
What do you mean by "dynamic" here? Dynamic suggests it is responding to something. If so, what? Can it be simplified as "Stretch"?

Reply:
changing to 'Non-Linear scale'